py3status/storage.py

```diff
@@ -90,7 +90,7 @@ def storage_set(self, module_name, key, value):
         if module_name not in self.data:
             self.data[module_name] = {}
         self.data[module_name][key] = value
-        ts = time()
+        ts = time.time()
```

Comment:
```suggestion
        ts = time.perf_counter()
```

Reply:
And here it goes to `storage`, so probably it will be stored on the disk and loaded by a new process. `time.perf_counter()` returns a float that is based on *some* time in the past, apparently the start of the process. So you cannot copy it between processes.